### ISSUE TITLE
Mykytea exits when multiple instances would be created

### ISSUE THREAD

**Issue by t2y** [chezou/Mykytea-python#4]
```python
>>> import Mykytea
>>> opt = "-model /usr/local/share/kytea/model.bin -deftag UNKNOWN!!"
>>> mk1 = Mykytea.Mykytea(opt)
>>> mk2 = Mykytea.Mykytea(opt)
```

Then, Python interpreter exits with below message.

```
Stray '-model' argument

kytea:
  KyTea, a word segmentation/pronunciation estimation tool

Analysis Options: 
  -model   The model file to use when analyzing text
  -nows    Don't do word segmentation (raw input cannot be accepted)
  -notags  Do only word segmentation, no tagging
  -notag   Skip the tag of the nth tag (n starts at 1)
  -nounk   Don't estimate the pronunciation of unknown words
  -wsconst Specifies character types to not be segmented (e.g. D for digits)
  -unkbeam The width of the beam to use in beam search for unknown words 
           (default 50, 0 for full search)
  -debug   The debugging level (0=silent, 1=simple, 2=detailed)
Format Options: 
...
```

I don't know what causes this happened, but it's related to  299ede443043a157681461034c939894243cc708 changes.

**Comment by chezou** [comment]
fixed 5a1b960

```
In [1]: import Mykytea
In [2]: opt = "-model /usr/local/share/kytea/model.bin"
In [3]: mk1 = Mykytea.Mykytea(opt)
In [4]: opt
Out[4]: '-model /usr/local/share/kytea/model.bin'
In [5]: mk2 = Mykytea.Mykytea(opt)
```

**Comment by chezou** [comment]
`strtok()` was broken `opt`.

To prepend braking input string, I intend to copy it but the code was wrong.
